src/shared/models/game/GameState.es6

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+//
+// GameState
+// Created by dpekar on 10/7/15.
+//
+
+const BaseModel = require('../BaseModel')
+const GameSetup = require('./GameSetup')
+// const BaseError = require('../error/BaseError')
+// const Player = require('./Player')
+// const Market = require('./Market')
+// const Tile = require ('./Tile')
+const _ = require('underscore')
+
+class GameState extends BaseModel {
+
+    constructor(users) {
+        super()
+        this.init(users)
+    }
+
+    init(users) {
+        this.setupPlayers(users)
+        this.setupTilePiles()
+        this.setupStartingTilesForPlayers()        
+        this.setupMarket()        
+        return this
+    }
+
+    getPlayers() {return this.get('players')}
+    getTilePiles() {return this.get('tilePiles')}
+    getMarket() {return this.get('market')}
+
+
+    placeTile(tile, player) {
+        // remove a tile from the market or basic market
```

Comment:
yes absolutely,  I have a design in mind for this.  I put it in the eng design doc last night.  I think every turn should get back a result object which is basically an array of effects that can be played out in order.  Each "effect" holds an stat (income), the change (+2), and the placement (farm on (2,2)) so it can all be animated on the front end step by step.